qa/setup_packages.py

```diff
@@ -375,7 +406,7 @@ def for_all_pckg(packages, fun):
 def get_remove_string(packages, cuda_version):
     """Creates pip remove string for given cuda version and package list"""
     # Remove only these which version we want to change
-    ret = for_all_pckg(packages, lambda pckg: pckg.get_name(cuda_version))
+    ret = for_all_pckg(packages, lambda pckg: pckg.get_uninstall_names(cuda_version))
```

Comment:
I know this is really unrelated, but in 
`def print_configs(cuda_version):` you don't iterate over all possible `idx`, thus printing only the first variant and ignoring the alias (as the idx=None -> 0 by default).

Reply:
This looks like it's only important when listing options by hand, so it doesn't matter much.

Reply:
Updated a bit.